packages/react-server-cli/src/compileClient.js

```diff
@@ -13,93 +13,6 @@ import callerDependency from "./callerDependency"
 // import {logging} from "react-server"
 // const logger = logging.getLogger(__LOGGER__);
 
-// compiles the routes file for browser clients using webpack.
-// returns a tuple of { compiler, serverRoutes }. compiler is a webpack compiler
-// that is ready to have run called, and serverRoutes is a promise that resolve to
-// a path to the transpiled server routes file path. The promise only resolves
-// once the compiler has been run. The file path returned from the promise
-// can be required and passed in to reactServer.middleware().
-// TODO: add options for sourcemaps.
-export default (opts = {}) => {
```

Comment:
I hear you; I think I was the original author of a few of those files in that format.

Another option is to convert functions in those files from `const foo = (bar) => { /* body */}` to `function foo(bar) {/* body */}` and rely on function hoisting, which I *think* would allow you to put the exports at the top in most cases (but maybe not all? haven't thought it through). Less ES6-y and relies on some hoisting magic, but maybe more readable? (Maybe?)